src/components/adaptive-cards/TextBlock/TextBlock.jsx

```diff
@@ -11,14 +11,17 @@ import {formatDateTime} from '../util';
  * @param {object} props  React props passed to the component
  * @param {object} props.data  Active cards definition
  * @param {string} [props.className]  Custom CSS class to apply
+ * @param {object} [props.style]  Custom style to apply
  * @returns {object} JSX of the component
  */
-export default function TextBlock({data, className}) {
-  const [cssClasses] = webexComponentClasses('adaptive-cards-text-block', className);
+export default function TextBlock({data, className, style}) {
+  const {maxLines} = data;
+  const [cssClasses] = webexComponentClasses('adaptive-cards-text-block', className, {maxLines});
```

Comment:
nice and short, but this will generate a camel-case classname, and we're generally using dash-separated class names, so `{'max-lines': maxLines}`